src/ui/public/chrome/directives/kbn_chrome.js

```diff
@@ -44,7 +47,9 @@ export default function (chrome, internals) {
         onRouteChange();
 
         // and some local values
-        chrome.httpActive = $http.pendingRequests;
+        $scope.$watch(() => $http.pendingRequests, (pendingHttpRequests) => {
+          chrome.httpActive = pendingHttpRequests.filter(request => !systemApi.isSystemApiRequest(request));
```

Comment:
I'd be more comfortable if we tweaked this implementation to preserve the existing behavior where `chrome.httpActive` is always the same array reference.  In otherwords, we should probably mutate the array rather than replace it.  It complicates this implementation, but it means that anything that relies on that behavior will continue to work.